config/v1/types_infrastructure.go

```diff
@@ -562,6 +573,21 @@ type BareMetalPlatformStatus struct {
 	// datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames
 	// to the nodes in the cluster.
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
+
+	// loadBalancerType defines if an user-managed or openshift-managed loadbalancer
+	// is to be used for the openshift API and Ingress endpoints.
+	// When set to OpenShiftManaged the static pods in charge of API and Ingress traffic load-balancing
+	// defined in the machine config operator will be deployed.
+	// When set to UserManaged these static pods will not be deployed and it is expected that
+	// the load balancer is configured out of band by the deployer.
+	// When omitted, this means no opinion and the platform is left to choose a reasonable default
+	// The current default value is OpenShiftManaged.
+	// +kubebuilder:default:="OpenShiftManaged"
```

Comment:
For compatibility with openapi gen, we need a `+default` tag with the same value please